packages/scripts/src/application.ts

```diff
@@ -113,15 +136,25 @@ export class Application {
         title,
         publicConfigsRoute,
         overrideConfig,
-    }: IRunOptions = {}): Promise<webpack.compilation.MultiStats> {
-        const engineConfig = await this.getEngineConfig();
-        if (engineConfig && engineConfig.require) {
-            await this.importModules(engineConfig.require);
+        external = false,
+        staticBuild = true,
+        withExternalFeatures,
+    }: IBuildCommandOptions = {}): Promise<webpack.compilation.MultiStats> {
+        const { require, externalFeatureDefinitions, externalFeaturesPath = join(this.basePath, 'node_modules') } =
+            (await this.getEngineConfig()) ?? {};
+        if (require) {
+            await this.importModules(require);
         }
+
+        if (external && !featureName) {
```

Comment:
I assume that we need to provide a feature name when building external, so that we will not create irrelevant mappings in the feature loaders